src/itemRadio/ItemRadio.tsx

```diff
@@ -105,7 +105,10 @@ class ItemRadio extends Component<ItemRadioProps> {
           rightTitle={data}
           rightTitleDisplay={TextDisplayType.SUBHEADERSTRONG}
           rightBody={dataInfo}
-          /* TODO: BBC-7415 fix a11y issue */
+          /* No a11y issue here
+            - The input is well wrapped with the label
+            - The linter can't access the complex components implementation
+          */
```

Comment:
Tested with keyboard nav + VoiceOver. Working well on my side.

Reply:
I think you should add a unit test to make sure that itemRadio always ouput a `<label />` with a very clear test description so that we don't break it.

Reply:
Done!